mailbox/server_conn.go

```diff
@@ -12,6 +12,24 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// ServerConnStatus is a description of the server's session state.
+type ServerConnStatus uint8
+
+const (
+	// ServerConnStatusNotConnected means that the server is currently not
+	// connected to a mailbox. This is either because the session is
+	// restarting or because the mailbox server is down.
+	ServerConnStatusNotConnected ServerConnStatus = 0
```

Comment:
Same here, drop both `ConnStatus` or just `Conn` in the constant names since it's probably quite obviously a connection?